stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -97,9 +97,14 @@ spec:
           mountPath: /fluent-bit/etc/parsers_custom.conf
           subPath: parsers.conf
 {{- end }}
+{{- if and .Values.backend.type "stackdriver" .Values.backend.stackdriver.google_service_credentials }}
+        - name: fluentbit-secret
```

Comment:
where is its respective volume that mounts this file from the host machine ?

Reply:
It is a secret mounted as a volume https://github.com/helm/charts/pull/15549/files#diff-f455fb5ea559681c3a3dd88acfd3e6aaR149